docroot/lib/views/switch_to_linux.from_windows_to_linux.php

```diff
@@ -29,30 +29,30 @@
+        <p><?php print _("Keeping your computer in shape over time doesn't need to be a struggle. One year on, GNU/Linux runs just as fast as the first day. And it may bring your old laptop back to life."); ?></p>
+        <p><?php print _("It should be noted that GNU/Linux lets you choose what you want and what you don't want. If you had a hard time getting rid of Microsoft Edge or adware and web browser toolbars, you'll find it refreshing."); ?></p>
 
         <h3 id="security"><?php print _("Security"); ?></h3>
         <p><?php print _("Forget expensive and restrictive anti-virus, anti-spyware, anti-anything and anti-everything. GNU/Linux is very, very secure."); ?></p>
-        <p><?php # translators: If you don't know of a link replacement in your languge, just leave the link as it is
-        print _("It is often difficult for Windows users to believe that there are <em>no spyware nor viruses</em> under GNU/Linux &mdash; <a href=\"http://linuxmafia.com/~rick/faq/#virus\" rel=\"external\">but it's true</a>. Malicious programs have a hard time doing anything at all in a well-built system."); ?></p>
-        <p><?php print _("Regular, easy software updates will keep everyone entirely safe."); ?></p>
+        <p><?php # translators: If you don't know of a link replacement in your language, just leave the link as it is
+        print _("It is often difficult for Windows users to believe that there are <em>no spyware nor viruses</em> under GNU/Linux&mdash;<a href=\"http://linuxmafia.com/~rick/faq/#virus\" rel=\"external\">but it's true</a>. Malicious programs have a hard time doing anything at all in a well-built system. No free (i.e. freedom respecting) application is &ldquo;phoning home&rdquo; if you didn't allow it."); ?></p>
+        <p><?php print _("Regular and easy software updates of GNU/Linux distributions keep everyone safe."); ?></p>
 
         <h3 id="support"><?php print _("Support"); ?></h3>
         <p><?php # translators: The first %s represents the URL to the "More" page and seconds %s represents the title of the page "More" in your language
         $p = _("GNU/Linux benefits from a great sense of community whose friendliness will surprise you. If you try to do something complicated but can't succeed, there are a lot of people around to help you out. Our \"<a href=\"%s\">%s</a>\" page gives a few pointers."); printf($p, $this->get_base_url('more'), _("More")); ?></p>
-        <p><?php print _("The companies behind the main GNU/Linux distributions, such as <a href=\"https://www.canonical.com/\" rel=\"external\">Canonical</a> and <a href=\"https://www.redhat.com/\" rel=\"external\">Red Hat</a>, also provide expert commercial support."); ?></p>
+        <p><?php print _("The companies behind the main GNU/Linux distributions, such as <a href=\"https://www.debian.org/consultants/\" rel=\"external\">Debian</a> and <a href=\"https://www.redhat.com/\" rel=\"external\">Red Hat</a>, also provide expert commercial support."); ?></p>
```

Comment:
I'm having seconds thoughts about removing Canonical here. Debian doesn't really have a company providing commercial support. Having a distribution backed by a large company like Red Hat or Canonical is what makes these distributions interesting to some users and companies. As long as we recommend Ubuntu as a distribution, we might as well recommend Canonical here. Is there a specific reason why you think we shouldn't recommend Canonical?